pkg/diagnose/deployment_metrics.go

```diff
@@ -0,0 +1,92 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package diagnose
+
+import (
+	"strings"
+
+	"github.com/submariner-io/admiral/pkg/reporter"
+	"github.com/submariner-io/subctl/pkg/cluster"
+)
+
+const (
+	gwCurlMetricsCommand = "curl -I -m 10 submariner-gateway-metrics.submariner-operator.svc.cluster.local:8080/metrics"
+	gnCurlMetricsCommand = "curl -I -m 10 submariner-globalnet-metrics.submariner-operator.svc.cluster.local:8081/metrics"
+)
+
+func CheckMetricsConfig(clusterInfo *cluster.Info, status reporter.Interface) bool {
```

Comment:
This is only accessed within the package.

```suggestion
func checkMetricsConfig(clusterInfo *cluster.Info, status reporter.Interface) bool {
```